src/Arcus.Template.WebApi/.template.config/template.json

```diff
@@ -22,6 +22,22 @@
         "value": false
       }
     },
+    "IfDebug": {
+      "type": "generated",
+      "generator": "constant",
+      "parameters": {
+        "value": "#if DEBUG"
+      },
+      "replaces": "//#if DEBUG"
+    },
+    "EndIf": {
+      "type": "generated",
+      "generator": "constant",
+      "parameters": {
+        "value": "#endif"
+      },
+      "replaces": "//#endif"
+    },
```

Comment:
Because we don't use env app settings; we can't use `env.IsDevelopment()`; which means directives (`#if`) has to be used.
The problem was that this is used in the template creation and will be removed at creation time.
The fix is to comment these out and uncomment them on creation (in `template.json`).

Reply:
I think that's a good suggestion so that they can still uncomment it if they want, what do you think?

Reply:
Yeah, it's not so 'clean' but it does the job well and besides from the devs that uses contributes to the project; no else sees the commented directives.

So, if you're do find this a good approach; then OK cool. 👌

Reply:
I think that's fine, yes, thanks!